testsuite/replay_verify.py

```diff
@@ -60,7 +60,7 @@
     # Skip tapos range
     (949_000_000, 954_000_000),
     (954_000_000, 969_000_000),
-    (969_000_000, sys.maxsize),
+    (969_000_000, 984_000_000),
```

Comment:
I think we can remove (392_000_000, 408_000_000),
and add a new range like (984_000_001, sys.maxsize)?

Reply:
I removed the oldest range and replaced it with a more recent range.

I still think we should specify an end version instead of using `sys.maxsize`. This started timing out recently and will likely happen again if we continue using `sys.maxsize`

Reply:
Yeah, on the other hand we do wanna make sure we're adding more transactions eventually

But there is a broader conversation here over whether or not this is the right, scalable way to ensure compatibility